spec/features/booking_manager_views_booking_requests_spec.rb

```diff
@@ -44,10 +60,15 @@ def then_they_see_the_administrative_location_choices
   end
 
   def and_there_are_booking_requests_for_their_location
-    create_list(:hackney_booking_request, 11)
+    @active_booking_requests = create_list(:hackney_booking_request, 8)
+    @active_booking_requests << create(:hackney_booking_request, name: 'Morty Smith')
+    @active_booking_requests << create(:hackney_booking_request, name: 'Jerry Smith')
+    @active_booking_requests << create(:booking_request, name: 'Dalston Dave',
```

Comment:
Can we put each argument on a new line and indented two characters? With the closing paren inline with the @ above.